--- src/main/java/io/github/dsheirer/gui/playlist/streaming/StreamingEditor.java
+++ src/main/java/io/github/dsheirer/gui/playlist/streaming/StreamingEditor.java
@@ -127,8 +127,9 @@
         getNewButton().getStyleClass().add("flat-button");
         getDeleteButton().getStyleClass().add("flat-button");
+        getCloneButton().getStyleClass().add("flat-button");
         getRefreshButton().getStyleClass().add("flat-button");

-        toolbar.getChildren().addAll(getNewButton(), getDeleteButton(), getRefreshButton());
+        toolbar.getChildren().addAll(getNewButton(), getDeleteButton(), getCloneButton(), getRefreshButton());

         VBox tableAndLabelBox = new VBox();
         VBox.setVgrow(getConfiguredBroadcastTableView(), Priority.ALWAYS);
