--- src/main/java/io/github/dsheirer/audio/broadcast/BroadcastFactory.java
+++ src/main/java/io/github/dsheirer/audio/broadcast/BroadcastFactory.java
@@ -37,6 +37,8 @@
 import io.github.dsheirer.audio.broadcast.zello.ZelloConfiguration;
 import io.github.dsheirer.audio.broadcast.zello.ZelloConsumerBroadcaster;
 import io.github.dsheirer.audio.broadcast.zello.ZelloConsumerConfiguration;
+import io.github.dsheirer.audio.broadcast.iamresponding.IAmRespondingBroadcaster;
+import io.github.dsheirer.audio.broadcast.iamresponding.IAmRespondingConfiguration;
 import io.github.dsheirer.audio.convert.ISilenceGenerator;
 import io.github.dsheirer.audio.convert.InputAudioFormat;
 import io.github.dsheirer.audio.convert.MP3Setting;
@@ -99,6 +101,8 @@
                 case ZELLO:
                     return new ZelloConsumerBroadcaster((ZelloConsumerConfiguration) configuration,
                             inputAudioFormat, mp3Setting, aliasModel);
+                case IAMRESPONDING:
+                    return new IAmRespondingBroadcaster((IAmRespondingConfiguration) configuration, aliasModel);
                 case UNKNOWN:
                 default:
                     mLog.info("Unrecognized broadcastAudio configuration: " + configuration.getBroadcastFormat().name());
@@ -148,6 +152,8 @@
                 return new ZelloConfiguration(format);
             case ZELLO:
                 return new ZelloConsumerConfiguration(format);
+            case IAMRESPONDING:
+                return new IAmRespondingConfiguration();
             case UNKNOWN:
             default:
                 mLog.info("Unrecognized broadcastAudio server type: " + serverType.name());
